starter-code/src/components/Countries.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Countries = (props) => {
+  return (
+    <div>
+      <p>Data</p>
+      {console.log(props.data)}
+      <p>{this.props.data}</p>
```

Comment:
Estas en un componente funcional, no necesitas this por que no estas en una clase que tenga un propiedad props, tienes una clase que recibe props por params.

Necesitas poner atencion a los errores, no veo mayor problema que ver el log que te arroja e ir a ver en el codigo por que y asi entregaste el lab, no todo es esperar a que te digan cuales son los problemas y menos en este tipo de entornos donde la misma harramienta te apoya un monton para que logres reparar los errores, trata de ponerle mas atencion y leerlos con cuidado.